client/ayon_cinema4d/startup/ayon_cinema4d.pyp

```diff
@@ -198,6 +204,11 @@ class ExperimentalTools(c4d.plugins.CommandData):
         return True
 
 
+class CONTEXTLABEL(c4d.plugins.CommandData):
```

Comment:
Why is this an all caps class name?

```suggestion
class ContextLabel(c4d.plugins.CommandData):
    """Stub menu entry to act as current context label"""
```

Reply:
Lazy copying and pasting. Will fix it.

Reply:
https://github.com/ynput/ayon-cinema4d/pull/12/commits/572ff86130aa40fdf5df9dd80d4047062d16dfc1